src/tools/tidy/src/deps.rs

```diff
@@ -35,11 +35,9 @@ const EXCEPTIONS: &[(&str, &str)] = &[
     ("ryu", "Apache-2.0 OR BSL-1.0"),   // rls/cargo/... (because of serde)
     ("bytesize", "Apache-2.0"),         // cargo
     ("im-rc", "MPL-2.0+"),              // cargo
-    ("constant_time_eq", "CC0-1.0"),    // rustfmt
     ("sized-chunks", "MPL-2.0+"),       // cargo via im-rc
     ("bitmaps", "MPL-2.0+"),            // cargo via im-rc
     ("crossbeam-queue", "MIT/Apache-2.0 AND BSD-2-Clause"), // rls via rayon
-    ("arrayref", "BSD-2-Clause"),       // cargo-miri/directories/.../rust-argon2 (redox)
```

Comment:
Could you also remove unused entries from `PERMITTED_DEPENDENCIES`?

Reply:
Erm, which ones? These are transitive dependencies, and don't seem to be covered by that list.

Reply:
I thought it also required listing all transitive dependencies. This indeed seems to not be the case.